msgraph/models.go

```diff
@@ -1685,3 +1685,11 @@ type EmployeeOrgData struct {
 	CostCenter *string `json:"costCenter,omitempty"`
 	Division   *string `json:"division,omitempty"`
 }
+
+type UserFlowAttribute struct {
+	ID                    *string `json:"id,omitempty"`
+	Description           *string `json:"description,omitempty"`
+	DisplayName           *string `json:"displayName,omitempty"`
+	UserFlowAttributeType *string `json:"userFlowAttributeType,omitempty"`
+	DataType              *string `json:"dataType,omitempty"`
```

Comment:
The documentation says that only "string", "boolean" and "int64" are supported.

Should we add these values here ?: https://github.com/manicminer/hamilton/blob/main/msgraph/valuetypes.go
I think this will make it a little less error prone.

Reply:
Sure! Added it in https://github.com/manicminer/hamilton/pull/182/commits/44848901e799dfbfc1ebd2a3eb42bca69bbbf79f